src/PrestaShopBundle/Model/Product/AdminModelAdapter.php

```diff
@@ -337,9 +337,8 @@ public function getModelData($form_data, $isMultiShopContext = false)
             $form_data['combinations'][$k]['attribute_unity'] = abs(
                 $this->floatParser->fromString($combination['attribute_unity'])
             );
-            $form_data['combinations'][$k]['attribute_quantity'] = abs(
-                $this->floatParser->fromString($combination['attribute_quantity'])
-            );
+            $form_data['combinations'][$k]['attribute_quantity'] = $this->floatParser->fromString($combination['attribute_quantity']);
```

Comment:
Does it make sense to have negative value for a quantity in this usecase? Just a naive question :)

Reply:
I think yes, maybe @PrestaShop/product-team can say more but with the option to allow out-of-stock ordering, you may want to have a negative value here

Reply:
When out of stock is enabled, product can be order even if there are no stock available. So to keep track of it, the product stock uses negative value. It can help the merchant to know how much they need to fabric or order to their supplier.

Reply:
negative quantity are very usefull to keep real track of product status. every serious ERP have this feature.